docs/src/content/en/docs/agents/sanitizing-input.mdx

```diff
@@ -176,59 +176,55 @@ Strategies available:
 - `block`: Reject content not in target language
 - `warn`: Log warning but allow content through
 
-## Applying Multiple Processors
+## Applying multiple processors
 
 You can chain multiple processors. They execute sequentially in the order they appear in the `inputProcessors` array. The output of one processor becomes the input for the next.
 
 **Order matters!** Generally, it's best practice to place text normalization first, security checks next, and content modification last.
```

Comment:
probably a nit, but I don't like the bold explanation here. isn't that what the callouts/alerts are for?